app/src/main/res/layout/vlc_player_interface.xml

```diff
@@ -58,114 +58,170 @@
 
     </FrameLayout>
 
-    <include
-        android:id="@+id/smNextUpPanel"
-        layout="@layout/small_upnext_panel"
-        android:layout_width="match_parent"
-        android:layout_height="75dp"
-        android:layout_gravity="bottom" />
+
+    <ProgressBar
```

Comment:
This is not present in the current version of the overlay, I think this file is broken. This ProgressBar was removed in https://github.com/jellyfin/jellyfin-androidtv/commit/ed01855a66309d1092b86a45e007aea0b2ff1422#diff-92d250046e8bf52735338f876b5b0009